spec/emerald_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'spec_helper'
+require 'tempfile'
+require 'helper/user_comms'
+include Emerald
+require 'emerald/parser'
+require 'emerald/compiler'
+
```

Comment:
I'm generally not in favour of contexts or before/after hooks, I think they make the tests less clear.